charts/incubator/plant-it/values.yaml

```diff
@@ -0,0 +1,75 @@
+image:
+  repository: msdeluise/plant-it-server
+  pullPolicy: IfNotPresent
+  tag: 0.4.3@sha256:a817573706ff221da8e03d339f84cd009ac94760d9d68682304bec9220e84128
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            MYSQL_HOST:
+              secretKeyRef:
+                name: cnpg-main-urls
+                key: host
+            MYSQL_PORT: "5432"
+            MYSQL_USERNAME: "{{ .Values.cnpg.main.user }}"
+            MYSQL_PSW: "{{ .Values.mariadb.password  }}"
+            MYSQL_DATABASE: "{{ .Values.cnpg.main.database }}"
+            MYSQL_ROOT_PASSWORD: {{  .Values.mariadb.rootPassword  }}
```

Comment:
```suggestion
            MYSQL_ROOT_PASSWORD: "{{  .Values.mariadb.rootPassword  }}"
```
Stop fucking around

Reply:
I was working on it. at the same time as you. And had already merged those changes. But couldnt merge proberly. So reverted mine